### ISSUE TITLE
`i-l` or `i-(l+1)`?

### ISSUE THREAD

**Issue by osheroff** [carlosholivan/Self-Similarity-Lag-Matrix#2]
Hi, 
Thanks for the excellent implementation of the paper.  It's been invaluable to me (just a coder, really) in trying to understand the mechanisms of the SSLM.  I've been using this implementation as a basis for integrating SSLMs into an MLS based neural net over here: https://github.com/mleimeister/SegmentationCNN/pull/9/files

The one question i had wrt to your implementation is here:
```
for i in range(padding_factor//p, distances.shape[0]): #iteration in columns of x_hat
    for l in range(padding_factor//p):
        epsilon[i,l] = np.quantile(np.concatenate((distances[i-l,:], distances[i,:])), kappa)
```

though I sadly am too ignorant to read the equations, this line appears incorrect to me: for `epsilon[0,0]` we'll be concatenating the same vector twice, won't we?  but I'm assuming what we actually wanted was a concatenation of the lag-values of the current frame and the frame it's being compared to, which should be `distances[i-(l+1)]`... no?

Thanks for your time.
Ben Osheroff

**Comment by carlosholivan** [comment]
Hi Ben,

First of all I'm sorry for the delayed response.

You are right because as you said, when we compute ````epsilon[0,0]```` we are concatenating the same vector twice, but if you want to train a neural network with the SSLMs and Mel Spectrograms I'm pretty sure that this change won't affect the accuracy of the model.

All the best,
Carlos.
